DataFormats/L1CaloTrigger/interface/CICADA.h

```diff
@@ -0,0 +1,10 @@
+#ifndef DataFormats_L1Trigger_CICADA_h
+#define DataFormats_L1Trigger_CICADA_h
+
+#include "DataFormats/L1Trigger/interface/BXVector.h"
+
+namespace l1t {
+  typedef BXVector<float> CICADABxCollection;
```

Comment:
Coming back to this question https://github.com/cms-sw/cmssw/pull/44222#discussion_r1514854288, maybe from a different angle, I wonder if `float` is a good long-term choice, as opposed to a dedicated `l1t:CICADA` class (however simple it may currently be).

Some points.
 - Today, `l1t::CICADA` would just be something like (in short) `class CICADA { float score; };`. Is there a possibility that in future one could use the available bits to extract more than a score (e.g. a score with reduced precision plus some different info from other bits) ? If so, a dedicated data format would be more future-proof, and it would minimise changes downstream in the future.
 - On the topic of downstream changes (and minimising them in the future), I was asked by Efe if the 'CICADA object' should be propagated to the machinery that HLT uses for "trigger objects" (see for example the discussion in https://github.com/cms-sw/cmssw/pull/45295). I don't think this is strictly necessary, but if it will/should be done, having a dedicated class `l1t:CICADA` would be better, imho. @cms-sw/hlt-l2 can correct me.
 - In https://github.com/cms-sw/cmssw/pull/45227, there is a different-but-similar use case, and in that case a dedicated class is being introduced (`AXOL1TLScore`).

I do not know if this (`float` vs `l1t:CICADA`) needs to be addressed in this PR or not, but it's worth bringing up imho.

Reply:
So this part of a dedicated data type is descoped?

Reply:
That would be my preference.

Reply:
OK!

Reply:
Okay, just had a chat with @eyigitba about the technical reasons for doing this. Strictly, I think it is agreed it is not needed for this PR, but may be more integral for "sophisticated" use at HLT (treating the CICADA score as an object in case we want to do that) than I was giving it credit for. I think there is also desire to centralize with AXO some of these ideas. I can take a look at this with a bit more priority in that case.